core/math/geometry.cpp

```diff
@@ -36,6 +36,7 @@
 #include "thirdparty/misc/clipper.hpp"
 #include "thirdparty/misc/triangulator.h"
 #define STB_RECT_PACK_IMPLEMENTATION
+#define STBRP_LARGE_RECTS
```

Comment:
Worth noting that stb_rect_pack 1.01 would make this the default behavior (we have 1.00): https://github.com/nothings/stb/blob/af1a5bc352164740c1cc1354942b1c6b72eacb8a/stb_rect_pack.h#L44